src/Docker/redis/Java/build.gradle

```diff
@@ -0,0 +1,39 @@
+apply plugin: 'java'
+
+repositories {
+    mavenCentral()
+}
+
+configurations {
+    redissonConfiguration 
+}
+
+ext {
+    defaultRedissonLibsDirectory = "tomcat/lib"
+    tomcatVersion = project.hasProperty('TomcatVersion') ? project.getProperty('TomcatVersion') : 'TOMCAT_10_1'
+    redissonLibsDirectory = project.hasProperty('RedissonLibsDirectory') ? project.getProperty('RedissonLibsDirectory') : 'defaultRedissonLibsDirectory'
```

Comment:
Look out with the 'defaultRedissonLibsDirectory', is a string, not the value of defaultRedissonLibsDirectory.

Reply:
Fixed.